query/src/configs/config_test.rs

```diff
@@ -63,6 +63,7 @@ log_level = \"INFO\"
 log_dir = \"./_logs\"
 
 [meta]
+meta_embedded_dir = \"./_meta_embedded\"
```

Comment:
Cool!
meta_embedded_dir is a test purpose path, it would be better to set it as a random uuid instead config it in the config file?
If so, We can improve it in the another PR